lgc/util/StartStopTimer.cpp

```diff
@@ -58,7 +63,23 @@ class StartStopTimer : public ModulePass {
   bool m_starting; // True to start the timer, false to stop it
 };
 
-char StartStopTimer::ID = 0;
+// Legacy PM wrapper
```

Comment:
Could we get away with not adding the wrapper? For example declare a new struct StartStopTimerPass of PassInfoMixin that would execute the same code in ::run() that the present runOnModule() is running, without modifying the old interface.

Reply:
We could, but that would imply code duplication, we'd need the same code for the two interfaces. I guess it is not really an issue for that specific pass because it is very small, but may be an issue for other bigger passes.

Reply:
Re code duplication: it could be avoided by making the core function static.
I am fine with the wrapper approach as it works too. It is a bit more code, but you already did the work.